src/flint/fmpz_mod_mat.jl

```diff
@@ -916,3 +916,38 @@ function matrix_space(R::ZZModRing, r::Int, c::Int; cached::Bool = true)
   ZZModMatrixSpace(R, r, c)
 end
 
+################################################################################
+#
+#  Kernel
+#
+################################################################################
+
+function AbstractAlgebra.Solve.kernel(M::ZZModMatrix; side::Symbol = :right)
```

Comment:
Should there be a check whether the modulus is prime here? Or can these rings never be fields?

Reply:
For now we ignore possible improvements.